apps/web/app/i/[shortUrl]/page.tsx

```diff
@@ -0,0 +1,11 @@
+import { redirect } from "next/navigation";
+import { getFullUrl } from "@formbricks/lib/services/urlshortener";
+
+export default async function ShortUrlPage({ params }) {
+  const fullUrl = await getFullUrl(params.shortUrl);
+
+  if (!fullUrl) redirect("/");
+  if (fullUrl) redirect(fullUrl);
```

Comment:
On line 7, we are alreday checking whether fullUrl exist, so we can just avoid rechecking it on next line

```
if (fullUrl) redirect(fullUrl);
  else {
    redirect("/");
  } 
```